doc/crypto/api/ops/pake.rst

```diff
@@ -1230,6 +1232,9 @@ J-PAKE does not assign roles to the participants, so it is not necessary to call
 J-PAKE requires both an application and a peer identity.
 If the peer identity provided to `psa_pake_set_peer()` does not match the data received from the peer, then the call to `psa_pake_input()` for the `PSA_PAKE_STEP_ZK_PROOF` step will fail with :code:`PSA_ERROR_INVALID_SIGNATURE`.
 
+J-PAKE does not use a context.
+A call to `psa_pake_set_context()` for a J-PAKE operation will fail with :code:`PSA_ERROR_INVALID_ARGUMENT`.
+
```

Comment:
I would prefer `PSA_ERROR_BAD_STATE` in this case. This is analog to `psa_cipher_set_iv()`, where `PSA_ERROR_BAD_STATE` shall be returned if the _cipher algorithm does not use an IV_.

Reply:
I think I agree. We should also change the text for `psa_pake_set_context()` - which currently describes `PSA_ERROR_INVALID_ARGUMENT` as being returned if the protocol does not support a context.

I do not think that the incompatibility (changing the specific error code) is an issue in practice here.